README.md

```diff
@@ -205,9 +205,43 @@ packages on the destination side can be ignored with
 
 By default, all subpackages of the specified packages are also ignored, but
 this can be disabled on an individual basis by adding
-`<ignoreSubpackages>false</ignoreSubpackages>` to the `<ignoreSourcePackage>`
+`<filterSubpackages>false</filterSubpackages>` to the `<ignoreSourcePackage>`
 or `<ignoreDestinationPackage>` element.
 
+### Allow only conflicts from certain packages
```

Comment:
Maybe this is a bit meta, but if I am understanding right it is not really that you want to "allow" conflicts in these packages, but rather this would be used to make missinglink report an error _only_ when the conflict happens in these packages, is that right? 

I wonder if using "allow" in the description and the property name would be confusing then, but at the same time I'm not sure what a better term would be - maybe something like "failOnConflictInPackages" or "target packages", what do you think?

Or maybe something along the lines of "packages to look for conflicts in" - its hard to think of a single term that describes this (unless I am just having a brain freeze)

Reply:
Rename to `target*`. lmk what you think!